README.md

```diff
@@ -1,53 +1,269 @@
+- [Dotnet Core SDK](https://dotnet.microsoft.com/download) must be installed to run this sample.
+- We recommend [VS Code](https://code.visualstudio.com/download) for running and debugging this cross-platform application.
 
 ## Setup
 
-Explain how to prepare the sample once the user clones or downloads the repository. The section should outline every step necessary to install dependencies and set up any settings (for example, API keys and output folders).
+Using a command line interface such as VS Code integrated terminal, locate the application directory. Then:
+
+1. Clone or download this repository:
+
+```console
+git clone https://github.com/Azure-Samples/ms-identity-javascript-angular-spa-aspnetcore-webapi.git
+```
+
+> [!NOTE]
+> Given that the name of the sample is quiet long, and so are the names of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
+
+1. Install .NET Core API dependencies:
+
+```console
+cd TodoListApplication/TodoListAPI
+dotnet restore
+```
+
+1. Trust development certificates:
+
+```console
+dotnet dev-certs https --clean
+dotnet dev-certs https --trust
+```
+
+1. Install Angular SPA dependencies:
+
+```console
+cd ../
+cd TodoListSPA
+npm install
+```
+
+Learn more about [HTTPS in .NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-3.1).
+
+1. Configure application parameters:
+
+See [Registration](#Registration) for details.
+
+
+## Registration
+
+### Register the sample application with your Azure Active Directory tenant
+
+There are two projects in this sample. Each needs to be separately registered in your Azure AD tenant. To register these projects, you can:
+
+- either follow the steps below for manual registration,
+- or use PowerShell scripts that:
+  - **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you. Note that this works for Visual Studio only.
+  - modify the Visual Studio projects' configuration files.
+
+<details>
+  <summary>Expand this section if you want to use this automation:</summary>
+
+1. On Windows, run PowerShell and navigate to the root of the cloned directory
+1. In PowerShell run:
+
+   ```PowerShell
+   Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+   ```
+
+1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+1. In PowerShell run:
+
+   ```PowerShell
+   cd .\AppCreationScripts\
+   .\Configure.ps1
+   ```
+
+   > Other ways of running the scripts are described in [App Creation Scripts](./AppCreationScripts/AppCreationScripts.md)
+   > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+1. Open the Visual Studio solution and click start to run the code.

```

Comment:
This line is probably not applicable for this sample

Reply:
Removed Visual Studio references. It works the same way in VS Code anyway.